qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageLocationAttributeTypesPage.java

```diff
@@ -33,6 +34,7 @@ public class ManageLocationAttributeTypesPage extends Page {
 	private static final By CONFIRM_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm button.confirm.right");
 	private static final By CANCEL_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm button.cancel");
 	private static final By CANCEL_DELETE_LOCATION_BUTTON = By.cssSelector("#purgeLocationAttributeTypeForm button.cancel");
+	private static final By RESTORE_LOCATION_ATTRIBUTE_TYPE = By.cssSelector("#adminui-restore-form-46 i.icon-reply.edit-action");
```

Comment:
Here we can't use the selector `#adminui-restore-form-46 i.icon-reply.edit-action` because the id `adminui-restore-form-46` varies with the number of rows in the table.

Reply:
@Piumal1999 what's your suggestion here?

Reply:
Maybe selecting the `i.icon-reply.edit-action` button in the last row?

Reply:
Thanks but with that selector, if the page has more one location attribute type, the test will fail because they would be so many items with the same selector.

Reply:
I mean we can use the selector `i.icon-reply.edit-action` with `clickOnLast()`.

`#adminui-restore-form-46 i.icon-reply.edit-action` selector won't work unless there are at least 46 rows in the table

Reply:
oh i get you, lemi do that